scania-digital-design-system/tegel
# fix(table): horizontal scroll broken

## **Describe pull-request**  
Reintroducing the fix made [here](https://github.com/scania-digital-design-system/tegel/pull/736) but only with the associated code that should have been. The old one got reverted [here](https://github.com/scania-digital-design-system/tegel/pull/764) due to the fact that unrelated code slipped in.

## **Issue Linking:**  
- **Jira:** [CDEP-3450](https://tegel.atlassian.net/browse/CDEP-3450)    

## **How to test**  
1. Go to Storybook
2. Check in Table
3. Enter value for horizontal scroll, should be possible to scroll horizontally in Table

## **Checklist before submission**

## **Suggested test steps**

## **Screenshots**  
_Include before/after screenshots for UI changes._

## **Additional context**  
_Add any other context or feedback requests about the pull-request here._


[CDEP-3450]: https://tegel.atlassian.net/browse/CDEP-3450?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
